test/e2e/framework/framework.go

```diff
@@ -456,10 +451,8 @@ func (f *Framework) CleanUp(ns string) error {
 func (f *Framework) CreateAndApplyAllocation(t *testing.T, flt *agonesv1.Fleet) *allocationv1.GameServerAllocation {
 	gsa := GetAllocation(flt)
 	gsa, err := f.AgonesClient.AllocationV1().GameServerAllocations(flt.ObjectMeta.Namespace).Create(context.Background(), gsa, metav1.CreateOptions{})
-	if !assert.NoError(t, err) {
-		assert.FailNow(t, "gameserverallocation could not be created")
-	}
-	assert.Equal(t, string(allocationv1.GameServerAllocationAllocated), string(gsa.Status.State))
+	require.NoError(t, err)
```

Comment:
Should this keep the message text as well ("gameserverallocation could not be created")?

Reply:
🤷🏻 I can put it back in if you feel strongly. Since we get a stacktrace you can find where the error is coming from, and it should be a rare failure, I wasn't too fussed about whether there was a message or not.